src/App.tsx

```diff
@@ -1,16 +1,26 @@
 import './App.scss';
+import React, { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
+
+export const App: React.FC = () => {
+  const [moviesList, setMoviesList] = useState(moviesFromServer);
+
+  const addNewMovie = (movie: Movie) => {
+    setMoviesList([...moviesList, movie]);
+  };
 
-export const App = () => {
   return (
     <div className="page">
       <div className="page-content">
-        <MoviesList movies={moviesFromServer} />
+        <MoviesList movies={moviesList} />
       </div>
       <div className="sidebar">
-        <NewMovie /* onAdd={(movie) => {}} */ />
+        <NewMovie
+          onAdd={(movie) => addNewMovie(movie)}
```

Comment:
Will works same, but less callbacks
```suggestion
          onAdd={addNewMovie}
```